src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -36,21 +43,42 @@
             marginRight: normalize(5),
             height: normalize(8)
           }}
+          width={isPreview ? 9 : 16}
+          height={isPreview ? 9 : 16}
         />
         {memberCount === undefined && props.isLoading ? (
           <Shimmer height={10} width={normalize(25)} />
         ) : (
-          <Text style={styles.domainMember(shouldDisplay)}>{memberCount}</Text>
+          <Text
+            style={[
+              styles.domainMember(shouldDisplay),
+              isPreview ? {fontSize: normalizeFontSize(9)} : {}
+            ]}>
+            {memberCount}
+          </Text>
         )}
-        <Text style={styles.domainMember(shouldDisplay)}> Members</Text>
+        <Text
+          style={[
+            styles.domainMember(shouldDisplay),
+            isPreview ? {fontSize: normalizeFontSize(9)} : {}
+          ]}>
+          {' '}
+          Members
+        </Text>
       </View>
       {props?.initialData?.memberCount === undefined && props.isLoading ? (
         <Shimmer height={10} width={normalize(60)} />
       ) : (
         isFollow &&
         !hideSeeMember && (
           <Pressable onPress={handlePress} style={{backgroundColor: COLORS.transparent}}>
-            <Text style={styles.seeMemberText(followType)} numberOfLines={1} ellipsizeMode="tail">
+            <Text
+              style={[
+                styles.seeMemberText(followType),
+                isPreview ? {fontSize: normalizeFontSize(9)} : {}
+              ]}
+              numberOfLines={1}
+              ellipsizeMode="tail">
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional rendering based on `isPreview` is repeated multiple times in this hunk. To improve readability and maintainability, consider creating a helper function that returns the appropriate style object based on the value of `isPreview`.

Here's an example:

```javascript
const getStyleBasedOnPreview = (defaultSize, previewSize) => {
  return isPreview ? {fontSize: normalizeFontSize(previewSize)} : {fontSize: normalizeFontSize(defaultSize)};
};
```

Then, you can use this function in your code like so:

```diff
- width={isPreview ? 9 : 16}
- height={isPreview ? 9 : 16}
+ width={getStyleBasedOnPreview(DEFAULT_WIDTH, 9)}
+ height={getStyleBasedOnPreview(DEFAULT_HEIGHT, 9)}
```

Again, replace `DEFAULT_WIDTH` and `DEFAULT_HEIGHT` with the default width and height values you want to use when `isPreview` is false.



<!-- This is an auto-generated comment by OSS CodeRabbit -->